core.html

```diff
@@ -137,6 +137,63 @@ <h3>Method 1: genesis doc</h3>
         is susceptible to attacks early in the DID's lifecycle. See <a target="_blank"
             href="https://github.com/openssi/peer-did-method-spec/issues/112">this github issue</a> for more discussion.
     </p>
+	
+    <h3>Method 3: multiple inception key without doc</h3>
+    <p>If <code>numalgo</code> == <code>3</code>, the generation mode is similar to Method 0 (and therefore also did:key)
+	with the ability to specify additional keys in the generated DID Document. This method is necessary when both an encryption
+	key and a signing key are required.</p>
+	<ul>
+		<li>Construct a peer:did as described in Method 0 with the desired encryption public key.</li>
+		<li>Construct a multibase encoded, multicodec-encoded form of the desired signing public key.</>
+		<li>Append the encoded signing key to the original DID using a period character `.` as a delimiter.</li>
+	</ul>
+
+	<pre class="example nohighlight" title="Example Multi-key Peer DID" id="multi-key-creation">
+		Encoded Encryption Key: z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH
+		Encoded Signing Key: zXwpBnMdCm1cLmKuzgESn29nqnonp1ioqrQMRHNsmjMyppzx8xB2pv7cw8q1PdDacSrdWE3dtB9f7Nxk886mdzNFoPtY
+		Method 3 peer DID: did:peer:3z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH.zXwpBnMdCm1cLmKuzgESn29nqnonp1ioqrQMRHNsmjMyppzx8xB2pv7cw8q1PdDacSrdWE3dtB9f7Nxk886mdzNFoPtY
+    </pre>	
+	
+	<p>When Resolving the peer DID into a DID Document, the process is similar to Method 0.</p>
+	<ul>
+		<li>Resolve the document as described in Method 0, using the portion of the DID prior to the delimiter.</li>
+		<li>Decode the key after the delimiter, and insert it into the PublicKey array, using this key as the id fragment.</li>
```

Comment:
I believe the "PublickKey" array is not longer part of the did core spec.  Need to decide where this goes.

Reply:
I was basing this off the did:key spec, which has not been updated. We'll need to make sure we align with the core spec.

Reply:
Looks like this part is missing now, but for future reference `publicKey` has been deprecated in favor of `verificationMethod`

Reply:
@llorllale has submitted pr #27 to update to `verificationMethod`. I'll try to merge both PRs together.